core/src/test/scala/org/apache/spark/scheduler/TaskSetManagerSuite.scala

```diff
@@ -635,6 +635,28 @@ class TaskSetManagerSuite extends FunSuite with LocalSparkContext with Logging {
     assert(manager.resourceOffer("execA", "host3", NO_PREF).get.index === 2)
   }
 
+  // regression test for SPARK-4939
+  test("node-local tasks should be scheduled right after process-local tasks finished") {
```

Comment:
Can you just change the name of the test to incorporate the JIRA -- so remove the comment and make it:
test("SPARK-4939: node-local...")